### PR TITLE
Upgrade Statinamic 0.7.0

### LINKED ISSUE #94: Update Statinamic
Update Statinamic to version 0.6.1+ once the minor changes are done as well

### PR DESCRIPTION
**Pull request by thangngoc89**
This PR will close #94

@MoOx @marcustisater

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,20 +5,40 @@
...
+            }
+          ]
+        ]
+      }
+    }
   },
   "eslintConfig": {
+    "parser": "babel-eslint",
     "extends": [
       "eslint-config-i-am-meticulous",
```
Comment (MoOx) [inline package.json]: you can remove this line, eslint-config-i-am-meticulous@2 /react include the index too.

File: package.json [hunk 2]
```diff
@@ -27,50 +47,61 @@
...
-    "react": "^0.14.6",
-    "react-dom": "^0.14.6",
+    "raw-loader": "^0.5.1",
+    "react": "^0.14.7",
+    "react-dom": "^0.14.7",
     "react-helmet": "^2.2.0",
-    "react-redux": "^2.1.2",
-    "react-router": "^1.0.3",
+    "react-redux": "^4.0.6",
+    "react-router": "^2.0.0-rc6",
```
Comment (MoOx) [inline package.json]: 2.0.0 is out (not rc anymore)

File: web_modules/utils/index.js [hunk 3]
```diff
@@ -1,40 +0,0 @@
...
-
-    // static build optimization
-    ...__PROD__ && {
-      collection:
-        minifyCollection(require("statinamic/lib/md-collection-loader/cache")),
-    },
-
-    pageComponents: layouts,
-  }
-)
```
Comment (MoOx) [inline web_modules/utils/index.js]: maybe you should ask @mxstbr for this change, he was the one who introduced this change (multiple things in one file).

File: package.json (hunk 2 above)
Comment (thangngoc89) [inline package.json]: Thanks.

**Comment by MoOx** [comment]
Except my comments, looks good. I will try to release 0.7 asap.

File: web_modules/utils/index.js (hunk 3 above)
Comment (thangngoc89) [inline web_modules/utils/index.js]: Alright. I can move it back if needed. It's up to you @mxstbr

File: package.json [hunk 4]
```diff
@@ -27,50 +47,61 @@
...
     "style-loader": "^0.12.4",
-    "stylelint": "^3.2.2",
-    "stylelint-config-standard": "^0.2.0",
+    "stylelint": "^4.0.0",
+    "stylelint-config-standard": "^2.0.0",
     "webpack": "^1.12.9",
     "whatwg-fetch": "^0.9.0"
   },
   "dependencies": {
+    "fs-extra": "^0.26.5",
```
Comment (thangngoc89) [inline package.json]: This again is a work around for https://github.com/MoOx/statinamic/issues/117

File: web_modules/utils/index.js (hunk 3 above)
Comment (marcustisater) [inline web_modules/utils/index.js]: @mxstbr is out skiing. I think we should move back for now.

**Comment by marcustisater** [comment]
Awesome, just merge it when you think it's good to go.

File: web_modules/utils/index.js (hunk 3 above)
Comment (mxstbr) [inline web_modules/utils/index.js]: Sorry I'm on vacation and skiing, so really slow to respond :sweat_smile: What's the issue here?

Comment (thangngoc89) [inline web_modules/utils/index.js]: I removed web_modules/utils/index amd broke it up into multiple files in web_modules/app/ .  What is your opinion about this?

Comment (mxstbr) [inline web_modules/utils/index.js]: I had some reasoning behind doing that, but I can't remember it now tbh. Looks fine to me.

**Comment by MoOx** [comment]
This branch needs to be rebased.

0.7.0 is out https://github.com/MoOx/statinamic/releases/tag/0.7.0

**Comment by thangngoc89** [comment]
@MoOx rebased.

**Comment by thangngoc89** [comment]
Almost forgot, added assets hashing

**Comment by marcustisater** [comment]
When running this on local I fetch an error with analytics? 

`ERROR in ./web_modules/GoogleAnalyticsTracker/index.js Module not found: Error: Cannot resolve module 'react-google-analytics`'

**Comment by thangngoc89** [comment]
@marcustisater did you tried `npm install` ?

**Comment by marcustisater** [comment]
Whoops, I totally forgot about that. Thanks.

Alright, everything is working here. @MoOx would you like to double check anything before we merge it into master?

**Comment by MoOx** [comment]
Lgtm.

**Pull request merged**